package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "notifyre-nodejs-sdk",
-  "version": "1.1.1",
+  "version": "1.1.2",

```

Comment:
Version should probably be 1.2.0 since these changes consists of new features and updates and not bugfixes

Reply:
ok will merge this to the new version branch and create a pr